gpii/node_modules/gpii-oauth2/gpii-oauth2-datastore/test/DbDataStoreTests.js

```diff
@@ -0,0 +1,1190 @@
+/**
+GPII DB Data Store Tests
+
+Copyright 2016 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+The research leading to these results has received funding from the European Union's
```

Comment:
I don't think it was - this comment section can be removed from this and subsequent work